packages/qgrid-ngx/src/lib/layer/layer.service.ts

```diff
@@ -25,6 +25,13 @@ export class LayerService {
 			const { nativeElement } = container.element;
 			nativeElement.parentElement.classList.add(`q-grid-layer-${name}`);
 
+		for (let el = nativeElement; !!el; el = el.parentElement) {
```

Comment:
consider to put the logic of finding q-grid element under the sep. function like getHost(): HTMLElement {}

Reply:
fixed